### PR TITLE
Base styles

### LINKED ISSUE #2: Kanban Test
I'm just using this to test GitHub projects, please ignore.

### PR DESCRIPTION
**Pull request by kognise**
I added some base styles. Also testing kanban. Closes #2

### CODE REVIEW

**Comment by kognise** [comment]
.

**Review by hyperupcall — APPROVED** [review]
Looks good

File: nc.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  </div>
+)
+`.trim())
+fs.writeFileSync(`src/components/${name}/${name}.module.css`, `
+.container {
+  
+}
+`.trim())
+const compIndex = fs.readFileSync('src/components/index.ts')
```
Comment (hyperupcall) [inline nc.js:23]: Not sure if this complies with your linting tools, but I would prefer if there was better space distribution within this file.

Comment (kognise) [reply nc.js:23]: Got it! WIll fix :D

Comment (kognise) [reply nc.js:23]: I made that in like 10 seconds lol

**Pull request merged**